### DISCUSSION TITLE
Feature request: filter tables by value in column

### DISCUSSION THREAD

**Discussion by mwchase** [marimo-team/marimo#948] [Ideas]
I'm pretty sure this isn't currently possible, but perhaps I missed part of the documentation.

I have some tabular data that includes a freeform text field; I'd like to be able to only look at cells that have values "similar to" some search string in that column. (Substring checking should be good enough, but some kind of fuzzy find would be excellent.)

**Comment by akshayka** [comment]
Thanks for the feature request! I've converted this into a GitHub issue: #950 

We can continue the conversation there
